autoload/SpaceVim/mapping.vim

```diff
@@ -143,7 +143,23 @@ endfunction
 function! SpaceVim#mapping#close_current_buffer() abort
   let buffers = get(g:, '_spacevim_list_buffers', [])
   let bn = bufnr('%')
-  let index = index(buffers, bn) 
+  if getbufvar(bn, '&modified', 0)
+    redraw!
+    echohl WarningMsg
+    echon 'current buffer contains unsaved modification, save file Y/N?'
```

Comment:
what will happen if a buffer is not opened from a file?

Reply:
It only check if the buffer is modified，even if it is not opened from  a file.

Reply:
yap, I am wondering if the file is not opened from a file, how to save it? does the user need to manually input a file name?